packages/pros-devices/src/smart/mod.rs

```diff
@@ -13,7 +13,7 @@
 //!
 //! Most devices can be created with a `new` function that generally takes a port number along with other
 //! device-specific parameters. All sensors are thread safe, however sensors can only be safely constructed
-//! using the [`Peripherals`] API.
+//! using the [`crate::peripherals`] API.
```

Comment:
```suggestion
//! using the [`peripherals`](crate::peripherals) API.
```

Reply:
I didn't link to `Peripherals` because there is also `AsyncPeripherals`.